Visual_Analysis.ipynb

```diff
@@ -454,9 +527,9 @@
    "formats": "ipynb,py:light"
   },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "sagemaker-distribution:Python",
```

Comment:
This might break things; now it somehow depends on CONDA.

Reply:
reverted